source/agora/node/BlockSerialize.d

```diff
@@ -18,8 +18,22 @@ import agora.common.Data;
 /// Type of delegate SerializeDg
 public alias SerializeDg = void delegate(scope const(ubyte)[]) pure nothrow @safe;
 
+/*******************************************************************************
+
+    Serialize the block struct and return it to ubyte[].
```

Comment:
```suggestion
    Serialize the block struct and return it as a ubyte[].
```

Reply:
suggested the Apply

Reply:
did you forget to push?

Reply:
How do you know? :) Thank you for the quick review.